views/info/products.ejs

```diff
@@ -0,0 +1,507 @@
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="/sources/bootstrap.css">
+    <title>SignUP</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css">
+    <link rel="stylesheet" href="/css/products.css">
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-sm-4 ">
+            <select class="form-select" name="product" id="product">
+                <option value="tüm">Tüm ürünler</option>
+                <option value="Akü">Akü</option>
+                <option value="Yağ">Araç Yağı</option>
+                <option value="Alarm">Alarm</option>
+                <option value="lastik">Lastik</option>
+                <option value="müzik">Müzik Sistemi</option>
+                <option value="cam">Cam Filmi</option>
+            </select>
+        </div>
+        <div class="col-sm-8 d-flex justify-content-end">
+            <button class="sepet"><i class="fas fa-shopping-cart" title="Sepetim"></i></button>
+            <button class="filter"><i class="fas fa-filter"></i></button>
+            <select class="form-select" name="arac" id="arac">
+                <option class="secenek" value="Yeni">Yeni Ürünler</option>
+                <option class="secenek" value="yüksek">Yüksek Puanlılar</option>
+                <option class="secenek" value="a'z">A'dan Z'ye</option>
+                <option class="secenek" value="teslimat">Hızlı Teslimat</option>
+                <option class="secenek " value="indirim">İndirimli</option>
+            </select>
+        </div>
+        <h1 class="d-flex justify-content-center  ">Ürünler</h1>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
```

Comment:
burdan `col-md-12` yerine `col-md-6` yapıp, her <div class="card">... ,için koyabilirsin
böyle her ürün boşluk olacak ve tablet için ikişer ikişer görünletecek.